### PR TITLE
New pcakages: rpi-libcamera:20230303 and rpi-libcamera-apps-1.1.2

### PR DESCRIPTION
**Pull request by strec007**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64

### CODE REVIEW

**Comment by strec007** [comment]
int conversion issue has been fixed in v.1.1.2, now all builds work. 

This package depends on rpi-libcamera which has not been integrated yet (PR  #42574). Is it OK to create a dependent pull request like this? Thank you.

**Comment by classabbyamp** [comment]
put everything in 1 pull request

**Comment by strec007** [comment]
> put everything in 1 pull request

In that case, this is the one pull request containing all changes. Including those from PR #42579, which I close. Thank you.

File: common/shlibs [hunk 1]
```diff
@@ -4238,3 +4238,5 @@ libdate-tz.so.3 chrono-date-3.0.1_1
 libayatana-ido3-0.4.so.0 ayatana-ido-0.9.2_1
 libayatana-indicator3.so.7 libayatana-indicator-0.9.3_1
 libayatana-appindicator3.so.1 libayatana-appindicator-0.5.91_1
+libcamera.so.0.0.4 rpi-libcamera-230303_1
+libcamera-base.so.0.0.4 rpi-libcamera-230303_1
```
Comment (paper42) [inline common/shlibs:4242]: These are the same sonames as what the libcamera package has, so this won't work correctly once #41263 is merged.

> PLEASE NOTE: when multiple packages provide the same SONAME, the first one (order top->bottom) is preferred over the next ones.

Comment (strec007) [reply common/shlibs:4242]: OK, so what's the solution of this? The user should be able to choose if they want libcamera, or rpi-libcamera (on Raspberry Pi). I understand that I should add the `conflicts` variable specifying libcamera package but how can we avoid the conflict in `common/shlibs`?

**Pull request closed without merging**